### PR TITLE
Initial Pychecker for OLA work

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: tools/rdm/DMXSender.py [hunk 1]
```diff
@@ -41,8 +41,7 @@ def __init__(self, ola_wrapper, universe, frame_rate, slot_count):
     if (frame_rate > 0 and slot_count > 0):
       logging.info('Sending %d FPS of DMX data with %d slots' %
                    (frame_rate, self._slot_count))
-      for i in xrange(0, self._slot_count):
-        self._data.append(0)
+      self._data.extend([0] * self._slot_count)
```
Comment (peternewman) [inline tools/rdm/DMXSender.py]: I've not tested this, but I believe I've got the syntax right.

**Comment by nomis52** [comment]
Resync please so this doesn't have all the ArtNet / E1.31 changes.

**Comment by peternewman** [comment]
This is down to where I branched from and will be resolved by https://github.com/OpenLightingProject/ola/pull/1001 .

**Comment by nomis52** [comment]
LGTM

File: python/examples/ola_devices.py [hunk 2]
```diff
@@ -21,6 +21,9 @@
 __author__ = 'nomis52@gmail.com (Simon Newton)'
 
 from ola.ClientWrapper import ClientWrapper
+from ola.OlaClient import RequestStatus
```
Comment (peternewman) [inline python/examples/ola_devices.py]: Are the changes here and below okay @nomis52 , before I apply them to the other files and we can start getting some of this merged in?

**Comment by nomis52** [comment]
LGTM so far

**Comment by peternewman** [comment]
Okay @nomis52 I think this chunk can go in if you're happy.

**Comment by peternewman** [comment]
This is just failing due to http://llvm.org/apt/llvm-snapshot.gpg.key .

**Comment by peternewman** [comment]
Now back and working.

**Comment by peternewman** [comment]
Merge @nomis52 ?

**Pull request merged**